### DISCUSSION TITLE
Has anyone integrated with stripe to do subscription?

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#1724] [Questions] [answered]
Hi I am trying to use stripe to process payments for users on a monthly subscription. Do I also need node.js to talk to stripe or can I do this with just Supabase? Examples are welcome. Thanks!

**Comment by inian** [comment, marked as answer]
Hi @milestones95, check out this repo which uses Stripe, Supabase and Next.js to build a SaaS application - https://github.com/vercel/nextjs-subscription-payments
